ComposeTesting/sample/src/main/java/com/microsoft/device/dualscreen/testing/sample/MainActivity.kt

```diff
@@ -36,6 +37,7 @@ class MainActivity : ComponentActivity() {
 @Composable
 fun ComposeTestingApp() {
     TwoPaneLayout(
+        paneMode = TwoPaneMode.HorizontalSingle,
```

Comment:
Why do we need this?

Reply:
It's not necessary, but I changed it to make it easier to write some sample tests based on posture! For instance, dual portrait should be two panes but dual landscape would still be the single pane view